sktime/markov/transition_counting.py

```diff
@@ -473,12 +473,21 @@ def __init__(self, lagtime: int, count_mode: str, physical_time='1 step', sparse
             |  'us',  'microsecond*'
             |  'ms',  'millisecond*'
             |  's',   'second*'
+        n_states : int, optional, default=None
+            Normally, the shape of the count matrix is a consequence of the number of encountered states in given
```

Comment:
does this mean that the default `None` case is when it's not artificially set?

Reply:
Yes

Reply:
i wouldn't say that's 100% clear from the docstring but it's the obvious inference so no need to change